.drone.yml

```diff
@@ -0,0 +1,57 @@
+---
+kind: pipeline
+type: docker
+name: testing
+
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+- name: vet
+  pull: always
+  image: golang:1.13
+  commands:
+  - make vet
```

Comment:
I feel that since we’re using an image without a patch that we should always be printing out the golang information.

Reply:
But most people including me don't care about the patch level.